bootstrapvz/common/assets/init.d/expand-root

```diff
@@ -13,8 +13,19 @@
 prog=$(basename $0)
 logger="logger -t $prog"
 
-device_path="/dev/xvda"
+growpart=$(which growpart) || {
+	$logger "growpart was not found on PATH. Unable to expand size."
+	exit 0
+}
 
+root_index="0"
+
+$growpart /dev/xvda $root_index || {
+	$logger "growpart failed. Unable to expand size."
+	exit 0
+}
+
+device_path="/dev/xvda$root_index"
```

Comment:
Is it possible to determine the disk and partition given the device path? I can `sed` just the device path into here, but `growpart` requires the disk and partition as arguments.